casa/OS/HostInfo.cc

```diff
@@ -124,7 +124,9 @@ Int HostInfo::numCPUs(bool use_aipsrc)				\
     static const String keyword("system.resources.cores");	\
     if ( use_aipsrc ) {						\
 	String value;						\
-	if ( Aipsrc::find(value, keyword) ) {			\
+	if (resources_cores > 0) {			\
+		return resources_cores;			\
```

Comment:
I'm not a big fan of these aligned slashes but if the original author went through the trouble to do so one might as well keep them aligned